src/Symfony/Component/Console/Tests/Output/ConsoleOutputTest.php

```diff
@@ -22,4 +23,19 @@ public function testConstructor()
         $this->assertEquals(Output::VERBOSITY_QUIET, $output->getVerbosity(), '__construct() takes the verbosity as its first argument');
         $this->assertSame($output->getFormatter(), $output->getErrorOutput()->getFormatter(), '__construct() takes a formatter or null as the third argument');
     }
+
+    public function testSetFormatter()
+    {
+        $output = new ConsoleOutput();
+        $outputFormatter = new OutputFormatter();
+        $output->setFormatter($outputFormatter);
+        $this->assertSame($outputFormatter, $output->getFormatter());
+    }
+
+    public function testSetVerbosity()
+    {
+        $output = new ConsoleOutput();
+        $output->setVerbosity(1);
+        $this->assertSame(1, $output->getVerbosity());
```

Comment:
in this case it shouldn't matter, because the verbosity should be exactly the one set using the setter method